code/modules/economy/account.dm

```diff
@@ -5,6 +5,7 @@
 	var/list/bank_cards = list()
 	var/add_to_accounts = TRUE
 	var/account_id
+	var/being_dumped = FALSE //pink levels are rising
```

Comment:
Instead of a var used by a single traitor device, you could handle this with a trait now that they're available to datums

Reply:
could you give an example of how to do this?

Reply:
`bank_account.add_trait(TRAIT_DUMPING, "crab_protocol")` to add,
`bank_account.remove_trait(TRAIT_DUMPING, "crab_protocol")` to remove
Trait name and trait source can be whatever you want as long as the add/remove proc uses the same ones.

To check if it's TRUE or not simply call `bank_account.has_trait(TRAIT_DUMPING)`

define `TRAIT_DUMPING `in the trait defines file somewhere in the unique traits

Reply:
That said, it's probably cleaner to keep a list of accounts being dumped and checking if the account is in the list, although it may be less efficient. As it is right now, multiple crab protocols might interfere with each other.